packages/namespaces/src/builder.ts

```diff
@@ -1,6 +1,7 @@
 import { DataFactory } from 'n3';
 import { NamedNode } from '@rdfjs/types';
 
+// TODO find a way to type the namespace
```

Comment:
I _think_ the solution to have types is to manually (or through some automation) create definitions from known/popular/needed namespaces. All other approaches have their own downsides and no perfect/good-enough solution have been found.

Reply:
I'm going to take a look at how CSS does it in https://github.com/CommunitySolidServer/CommunitySolidServer/blob/main/src/util/Vocabularies.ts

Reply:
Tracked in #53